### PR TITLE
todo.sh: add page

### PR DESCRIPTION
**Pull request by BuonOmo**
Document https://github.com/todotxt/todo.txt-cli, version 2.12.0

- [x] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [x] The page(s) have at most 8 examples.
- [x] The page description(s) have links to documentation or a homepage.
- [x] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- **Version of the command being documented (if known): 2.12**

### CODE REVIEW

File: pages/common/todo.sh.md [hunk 1]
```diff
@@ -0,0 +1,28 @@
+# todo.sh
+
+> A simple and extensible shell script for managing your todo.txt file.
```
Comment (kbdharun) [inline pages/common/todo.sh.md:3]:
```suggestion
> Simple and extensible shell script for managing your `todo.txt` file.
```

File: pages/common/todo.sh.md [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+> More information: <https://github.com/todotxt/todo.txt-cli>.
+
+- List every items:
```
Comment (kbdharun) [inline pages/common/todo.sh.md:6]:
```suggestion
- List every item:
```

IG we can use a singular noun here.

**Review by kbdharun — COMMENTED** [review]
Thanks for your contribution. I have some suggestions for this page.

**Review by kbdharun — APPROVED** [review]

File: pages/common/todo.sh.md [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+
+`todo.sh rm {{item_no}}`
+
+- Set an item's [pri]ority (A-Z):
+
+`todo.sh pri {{item_no}} {{priority}}`
+
+- Replace an item:
+
+`todo.sh replace {{item_no}} '{{new_description}}'`
```
Comment (acuteenvy) [inline pages/common/todo.sh.md:28]:
```suggestion
`todo.sh replace {{item_no}} {{new_description}}`
```

**Review by acuteenvy — APPROVED** [review]

File: pages/common/todo.sh.md (hunk 3 above)
Comment (BuonOmo) [reply pages/common/todo.sh.md:28]: @acuteenvy the todo.sh documentation encourage using strings for the descriptions. Why do you think I should remove it ?

Comment (acuteenvy) [reply pages/common/todo.sh.md:28]: The placeholder contains no spaces, that's why I thought about removing quotes. But there's a lot of options that can be put inside the string (`@context`, etc.), so never mind.

**Pull request merged**